test/sanity/range.wtest

```diff
@@ -77,7 +77,7 @@ describe "Range Test Case"{
 
 	test "map" {
 		 
-		var range = 0 .. 5 
+		var range = 0 .. 5
```

Comment:
ok, weird I didn't delete that character !

Reply:
I must have done it out of compulsion XD